docker/Dockerfile.devel

```diff
@@ -66,12 +67,17 @@ WORKDIR /dali
 
 COPY . .
 
+ARG TRITON_VERSION
```

Comment:
This is already defined in line 28. Am I missing something?

Reply:
From docker docs:

> An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage:
```
ARG VERSION=latest
FROM busybox:$VERSION
ARG VERSION
RUN echo $VERSION > image_version
```

Reply:
I see, that explain many issues I had in the past when using ARGs :). Thanks